sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/tests/Implementation/BlobServiceTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Implementation;
+using Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Interface;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Tests.Implementation
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.Self)]
+    public class BlobServiceTests
+    {
+        private Mock<ILogger>? _loggerMock;
+        private BlobService? _blobService;
+
+        [SetUp]
+        public void Setup()
+        {
+            _loggerMock = new Mock<ILogger>();
+            _blobService = new BlobService(_loggerMock.Object);
+        }
+        [Test]
+        public async Task UploadBufferAsync_WithException_LogsError()
+        {
+            string uri = "invalid_uri";
+            string buffer = "Test buffer";
+            string fileRelativePath = "test/path";
+
+            await _blobService!.UploadBufferAsync(uri, buffer, fileRelativePath);
+
+            _loggerMock!.Verify(logger => logger.Error(It.IsAny<string>()), Times.Once);
+        }
+
+        [Test]
+        public void GetCloudFilePath_WithValidParameters_ReturnsCorrectPath()
+        {
+            string uri = "https://example.com/container";
+            string fileRelativePath = "test/path";
```

Comment:
we can add few test with invalid fileRelativePath

Reply:
we are not handling this in getCloudFilePath function.